sdk/communication/Azure.Communication.JobRouter/tests/RouterClients/DistributionPolicyLiveTests.cs

```diff
@@ -0,0 +1,257 @@
+            Assert.IsFalse(bestWorkerModeDistributionPolicy.Mode.BypassSelectors);
+
+            var paramSelectors = ((BestWorkerMode)bestWorkerModeDistributionPolicy.Mode).ScoringRuleOptions
+                .ScoringParameters;
+            Assert.AreEqual(1, paramSelectors.Count);
+            Assert.AreEqual(ScoringRuleParameterSelector.WorkerSelectors, paramSelectors.First());
+
+            var scoringRule = ((BestWorkerMode)bestWorkerModeDistributionPolicy.Mode).ScoringRule;
+            Assert.NotNull(scoringRule);
+            var azureFuncScoringRule = (FunctionRule)scoringRule;
+            // Assert.AreEqual("https://my.function.app/api/myfunction?code=Kg==", azureFuncScoringRule.FunctionAppUrl);
+            Assert.IsNotNull(azureFuncScoringRule.Credential);
+
+            if (Mode != RecordedTestMode.Playback)
+            {
+                // any value will be sanitized when recordings are saved
+                Assert.AreEqual("MyAppKey", azureFuncScoringRule.Credential.AppKey);
+                Assert.IsTrue(string.IsNullOrWhiteSpace(azureFuncScoringRule.Credential.FunctionKey));
+            }
+
+            Assert.AreEqual("MyClientId", azureFuncScoringRule.Credential.ClientId);
+
+            bestWorkerModeDistributionPolicyResponse = await routerClient.UpdateDistributionPolicyAsync(
+                new UpdateDistributionPolicyOptions(bestWorkerModeDistributionPolicyId)
+                {
+                    Mode = new BestWorkerMode(
+                        new FunctionRule( new Uri("https://my.function.app/api/myfunction?code=Kg=="), new FunctionRuleCredential("MyKey")),
+                        new List<ScoringRuleParameterSelector>()
+                        {
+                            ScoringRuleParameterSelector.WorkerSelectors
+                        },
+                        minConcurrentOffers: 1, maxConcurrentOffers: 2),
+                });
+
+            Assert.NotNull(bestWorkerModeDistributionPolicyResponse.Value);
+
+            bestWorkerModeDistributionPolicy = bestWorkerModeDistributionPolicyResponse.Value;
+            Assert.AreEqual(1, bestWorkerModeDistributionPolicy.Mode.MinConcurrentOffers);
+            Assert.AreEqual(2, bestWorkerModeDistributionPolicy.Mode.MaxConcurrentOffers);
+            Assert.IsFalse(bestWorkerModeDistributionPolicy.Mode.BypassSelectors);
+
+            paramSelectors = ((BestWorkerMode)bestWorkerModeDistributionPolicy.Mode).ScoringRuleOptions
+                .ScoringParameters;
+            Assert.AreEqual(1, paramSelectors.Count);
+            Assert.AreEqual(ScoringRuleParameterSelector.WorkerSelectors, paramSelectors.First());
+
+            scoringRule = ((BestWorkerMode)bestWorkerModeDistributionPolicy.Mode).ScoringRule;
+            Assert.NotNull(scoringRule);
+            azureFuncScoringRule = (FunctionRule)scoringRule;
+            // Assert.AreEqual("https://my.function.app/api/myfunction?code=Kg==", azureFuncScoringRule.FunctionAppUrl);
```

Comment:
nit:  If this is unused, we should consider removing it.

Reply:
This was a miss. Have been uncommented